### PR TITLE
Adding support for filtering on several nodes

### PR DESCRIPTION
**Pull request by JockeCK**
As we discussed the default behavior will be to be able to select several values. 

An small change is needed in omnitable to pass along the three new values. I will create an PR for that.

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
+import { memoooize } from '@neovici/cosmoz-utils/memoize';
 
-const
+const /**
+	 * Get a language sensitive string comparison object.
+	 * @param {string} locale Current locale.
+	 * @returns {object} Collator.
+	 */
+	_makeCollator = (locale) => new Intl.Collator(locale || undefined),
+	makeCollator = memoooize(_makeCollator),
```
Comment (megheaiulian) [inline utils.js:9]: Do we need to use `memoize` instead ? `memoooize` is mean to be used by function with 3 params ? 
We also should not memoize in the global scope as that memory cannot be released  when we remove the column or omnitable itself.

Comment (JockeCK) [reply utils.js:9]: or should I skip it? Just do makeCollator = (locale) => new Intl.Collator(locale || undefined)?

Comment (megheaiulian) [reply utils.js:9]: its fine with memoize.

**Review by cristinecula — APPROVED** [review]
LGTM

**Pull request merged**